week-4/InClass/A-objects-intro/exercise-part-1.js

```diff
@@ -6,4 +6,28 @@ can describe with a JavaScript object
 Assign each of them to a separate variable
 
 */
-
+let hoodie = {
+    brand: "Puma",
+    colours: ["purple", "black"],
+    yearsOwned: 2
+  }
+  
+  let book = {
+    author: "Daphne Du Maurier",
+    yearOfRelease = "1938",
+    protagonists: ["Mrs de Winter", "Maxim de Winter"],
+    isPreOwned: true
+  }
+  
+  let nintendoSwitch = {
+    joyConColours = ["blue", "red"],
+    noOfGames = 5,
+    newestGame = "Animal Crossing: New Horizons"

```

Comment:
Same as above, double check the "key: value" pairs of your `nintendoSwitch` object.